server/src/main/scala/email/GovNotifyEmail.scala

```diff
@@ -0,0 +1,47 @@
+package email
+
+
+import controllers.application.NegativeFeedbackData
+import org.slf4j.{Logger, LoggerFactory}
+
+import java.util
+import scala.collection.JavaConverters.mapAsJavaMapConverter
+import scala.util.Try
+import uk.gov.service.notify.{NotificationClient, SendEmailResponse}
+
+
+class GovNotifyEmail(apiKey: String) {
+
+  val log: Logger = LoggerFactory.getLogger(getClass)
+
+  val client = new NotificationClient(apiKey)
+
+  def positivePersonalisationData(url:String): util.Map[String, String] = {
+    Map(
+      "url" -> url
+    ).asJava
+  }
+
+  def negativePersonalisationData(feedbackData: NegativeFeedbackData): util.Map[String, String] = {
+    val contactMe = if (feedbackData.contactMe) s"The user ${feedbackData.feedbackUserEmail} is happy to be contacted." else s"The user ${feedbackData.feedbackUserEmail} is not happy to be contacted."
```

Comment:
I would rephrase '..not happy..' to something like 'The user X would not like to be contacted'